servicebroker/config/settings.json

```diff
@@ -1,5 +1,7 @@
 {
   "port": 8080,
   "user": "user",
```

Comment:
Michael, how about this time? I see the branch has conflicts now. Should I raise another 2 PR instead?